docs/quickstart.md

```diff
@@ -62,6 +62,22 @@ sudo apt-get install python3.7-dev
 
 - Windows users: install <a href="https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019" target="_blank">tools for Visual Studio</a>.
 
+- To install Go, follow the
+ official guide, depending on your platform: 
+ [https://golang.org/doc/install](https://golang.org/doc/install)
+
+- Python is already included by default on 
```

Comment:
Higher up there is already something on Ubuntu/Debian. Is that 3.7 specific?

Reply:
the thing above is about Python headers, that are not included by default. Here we are only talking about the interpreter.

Reply:
True